website/client/components/challenges/challengeModal.vue

```diff
@@ -47,10 +47,10 @@
         label
           strong(v-once) {{$t('endDate')}}
         b-form-input.end-date-input
-      .form-group(v-if='creating')
+      .form-group
```

Comment:
@MathWhiz this part of the form shouldn't be shown at all while a challenge is being edited, why didn't the v-if work already?

Reply:
Oh, I was just testing to see if the :disabled property worked, and forgot to reset it.